var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -12,44 +12,88 @@ class Neuron(CMakePackage):
     NEURON is a simulation environment for modeling individual and networks of
     neurons. NEURON models individual neurons via the use of sections that are
     automatically subdivided into individual compartments, instead of
-    requiring the user to manually create compartments.
+    requiring the user to manually create compartments. The primary scripting
+    language is HOC but a Python interface is also available.
     """
 
     homepage = "https://www.neuron.yale.edu/"
-    url = "https://neuron.yale.edu/ftp/neuron/versions/v7.7/nrn-7.7.tar.gz"
+    url = "https://github.com/neuronsimulator/nrn/releases/download/8.2.3/nrn-full-src-package-8.2.3.tar.gz"
     git = "https://github.com/neuronsimulator/nrn"
-    maintainers = ["pramodk", "nrnhines", "iomaganaris", "alexsavulescu"]
+    maintainers("pramodk", "nrnhines", "iomaganaris", "ohm314", "matz-e")
 
-    version("develop", branch="master", submodules="True")
-    version("8.0.0", tag="8.0.0", submodules="True")
-    version("7.8.2", tag="7.8.2", submodules="True")
-    version("7.8.1", tag="7.8.1", submodules="True")
+    version("develop", branch="master")
```

Comment:
Upstream will tell you this should be version `"master"` last I recall.

Reply:
yeah, I remember and it's a bit annoying to have that changed. I guess there will be some more changes from the upstream review. After that, I pull the upstream recipe into our fork.

Reply:
I'd move this to our `patch` repo.

Reply:
done!